docs/source/conf.py

```diff
@@ -441,15 +446,552 @@ def check_timeout_and_api_kwargs_presence(obj: object) -> int:
+
+                method_link = self._generate_link_to_method(method, klass)
+
+                # If we do get_args() when return annotation has one single class
+                # (e.g. <class 'telegram._message.Message'>), get_args() returns an empty tuple.
+                if not returned_classes:
+                    self._resolve_arg_and_add_link_to_method(
+                        arg=ret_annot,
+                        dict_of_methods_for_class=methods_for_class,
+                        link_to_method=method_link,
+                    )
+                    continue
+
+                for ret_cls in returned_classes:
+                    self._resolve_arg_and_add_link_to_method(
+                        arg=ret_cls,
+                        dict_of_methods_for_class=methods_for_class,
+                        link_to_method=method_link,
+                    )
+
+        return self._generate_admonitions(methods_for_class, admonition_type="returned_in")
+
+    def _create_use_in(self) -> Dict[type, str]:
+        """Creates a dictionary with 'Use in' admonitions for classes whose instances are
+        accepted as arguments for Bot's and ApplicationBuilder's methods.
+        """
+
+        # Generate a mapping of classes to links to Bot methods which accept them as arguments,
+        # i.e. {<class 'telegram._inline.inlinequeryresult.InlineQueryResult'>:
+        # {:meth:`telegram.Bot.answer_inline_query`, ...}}
+        methods_for_class = defaultdict(set)  # using set because there can be repetitions
+
+        for klass, relevant_methods_for_class in self.METHODS_FOR_BOT_AND_APPBUILDER.items():
+            for method in relevant_methods_for_class:
+                method_link = self._generate_link_to_method(method, klass)
+
+                sig = inspect.signature(getattr(klass, method))
+                parameters = sig.parameters
+
+                for param in parameters.values():
+                    resolve_and_add = partial(
+                        self._resolve_arg_and_add_link_to_method,
+                        dict_of_methods_for_class=methods_for_class,
+                        link_to_method=method_link,
+                        # we will also add "Use in" admonitions to subclasses (if applicable)
+                        include_subclasses_if_applicable=True,
+                    )
+                    annotation = param.annotation
+
+                    if isinstance(annotation, (type, str)):
+                        resolve_and_add(arg=annotation)
+
+                    elif isinstance(annotation, typing.TypeVar):
+                        # gets access to the "bound=..." parameter
+                        resolve_and_add(arg=annotation.__bound__)
+
+                    elif typing.get_origin(annotation) in (
+                        dict,
+                        type,
+                        collections.Sequence,
+                        collections.Callable,
```

Comment:
```suggestion
                        collections.abc.Sequence,
                        collections.abc.Callable,
```

those have been moved for quite a while and `collections.{Sequence, Callbale}` have been removed in 3.9 or so IIRC

Reply:
I guess `Coroutine` as well then. Done.
(It works on Python 3.9 but still)